maestro-studio/web/.nvmrc

```diff
@@ -1 +1 @@
-v16.14.0
+v20.19.4
```

Comment:
Umm, I’m not fully sure about updating all these npm packages and the Node version. My main concern is what happens if someone runs into issues on the web studio. I don’t think we should be solving those, which is why I feel we should just leave it as is.

Reply:
That bump is only for local developers. GitHub Actions already builds with whatever is local.

Reply:
And they're already getting an app built in Node v20 (or maybe newer - unsure what's on a GitHub runner currently)